packages/x-date-pickers/src/DesktopDatePicker/DesktopDatePicker.tsx

```diff
@@ -116,6 +116,7 @@ DesktopDatePicker.propTypes = {
   /**
    * Clear text message.
    * @default 'Clear'
+   * @deprecated
```

Comment:
Maybe add a message on the deprecation for each prop

```suggestion
   * @deprecated Use the `localeText` prop of `LocalizationProvider` instead, see [link to the doc]
```